src/openms/source/CHEMISTRY/EnzymaticDigestionLogModel.cpp

```diff
@@ -161,9 +161,10 @@ namespace OpenMS
   {
     SignedSize count = 1;
     AASequence::ConstIterator iterator = protein.begin();
-    while (nextCleavageSite_(protein, iterator), iterator != protein.end())
+    while (iterator != protein.end())
```

Comment:
I think this changes the code flow for an empty protein, makes it correct now. may also be good to test

Reply:
flow is probably correct now but count is 1 instead of zero for empty proteins